src/Akka.Hosting/AkkaHostedService.cs

```diff
@@ -72,6 +72,9 @@ async Task TerminationHook()
             catch (Exception ex)
             {
                 Logger.Log(LogLevel.Critical, ex, "Unable to start AkkaHostedService - shutting down application");
+                
+                // resolve https://github.com/akkadotnet/Akka.Hosting/issues/470 - never allow failures to be silent
+                Console.WriteLine($"Unable to start AkkaHostedService due to [{ex}] - shutting down application");
```

Comment:
Ensure that this gets logged regardless of how the MSFT.EXT.Logger inputs have been configured

Reply:
I think for exception, it should always be added at the end, because the stack trace will screw up the string formatting.

Reply:
Or we can change that to "due to [{ex.Message}] - shutting down application.\nCause: {ex}"

Reply:
or maybe we should just re-throw the exception as a new exception and let MS.EXT.Hosting deal with it.